src/Toolkit/A.php

```diff
@@ -726,4 +726,53 @@ public static function wrap($array = null): array
             return $array;
         }
     }
+
+    /**
+     * Filter the array using the given callback
+     * using both value and key
+     *
+     * @param array $array
+     * @param callable $callback
+     * @return array
+     */
+    public static function where(array $array, callable $callback): array
+    {
+        return array_filter($array, $callback, ARRAY_FILTER_USE_BOTH);
+    }
+
+    /**
+     * Filter the array, using only the given key or array of keys
+     *
+     * @param array $array
+     * @param int|string|array $keys
+     * @return array
+     */
+    public static function only(array $array, $keys): array
```

Comment:
I find these names very vague and not easy to direct understand what they do. Maybe something more like `A::filterKeys($array, $keys)` or `A::withKeys()` and `A::withoutKeys()`?

Reply:
Perhaps `A::onlyKeys()` and `A::withoutKeys()`? Does that sound good?

Reply:
Damn, the more I think about it, the more does `A::onlyKeys()` also suggest something else. Naming is really hard, that's why I reused names I used for my personal function previously :D

Reply:
🤦‍♂️ I realised I'm stupid, and `A::only()` is already in the toolkit, except called `A::get()` - if used with an array of keys. So to be in line with collections, which also have `->get()` and `->without()`, I renamed this to `A::without()`